mobile/src/services/user-details-api/user-details-api.service.ts

```diff
@@ -43,6 +44,26 @@ class UserDetailsApi {
       },
     );
   }
+
+  public updateUserAvatar({
+    file,
+    userId,
+  }: UserDetailsUpdateAvatarRequestDto): Promise<UserDetailsResponseDto> {
+    const { uri, type, fileName: name } = file;
+    const formData = new FormData();
+    formData.append('file', { uri, type, name } as unknown as string);
```

Comment:
Blob will be more appropriate type
```suggestion
    formData.append('file', { uri, type, name } as unknown as Blob);
```